htsget.md

```diff
@@ -102,34 +124,50 @@ _optional string_
 </td><td>
 Request read data in this format. Default: BAM. Allowed values: BAM,CRAM.
 
-Server replies with HTTP status 409 if the requested format is not supported.
+The server SHOULD reply with an `UnsupportedFormat` error if the requested format is not supported.
```

Comment:
Well, it's a choice between SHOULD or MUST here:

```
SHOULD   This word, or the adjective "RECOMMENDED", mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course.
```

```
MUST   This word, or the terms "REQUIRED" or "SHALL", mean that the
   definition is an absolute requirement of the specification.
```
I don't think it's an absolute requirement of the specification here that the server reply with the right error. If they respond with a more generic error, then the client won't be able to take the appropriate corrective action.

I'm not sure though, this is just my interpretation of how these things are specified.